Automattic/jetpack
# Sharing: Use add_filter instead of add_action

## Proposed changes:

Update a call to the `plugin_action_links_{$plugin_file}` hook in the Sharing module.

### Other information:


## Jetpack product discussion

* N/A

## Does this pull request change what data or activity we track or use?

* No

## Testing instructions:

* Go to Jetpack > Settings > Sharing and enable the Sharing buttons.
* Go to the Plugins menu
    * Nothing should change on the page. The `sharedaddy/sharedaddy.php` plugin doesn't exist anymore, so that code should not be used.
